code/style-illustration.css

```diff
@@ -0,0 +1,181 @@
+body {
+    background-color: rgb(169, 203, 206);
+    color: rgb(232, 250, 250);
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+  }
+  
+  h1 {
+    font-size: 64px;
+    margin: 0;
+    font-family: 'Neucha', cursive;
+}
+  
+  h2 {
+    font-size: 24px;
+    font-family: 'Amatic SC', cursive;
+  }
+  
+  h3 {
+    font-size: 24px;
+    padding: 0 8px;
+    margin-top: 16px;
+    margin-bottom: -16px;
+  }
+  
+  h4 {
+    font-size: 16px;
+    padding: 8px;
+  }
+  
+  p {
+    font-size: 14px;
+    margin: 0;
+  }
+  
+  /*** HEADER ***/
+  
+  header {
+    background-color: rgb(66, 116, 113);
+    height: 100px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 20px;
+    border-radius: 20px;
+    font-size: 24px;
+    font-family:  'Zen Kurenaido', sans-serif;
+  }
+  
+  header img {
+    height: 100%;
+  }
+  
+  nav a {
+    color: rgb(255, 255, 255);
```

Comment:
Sometimes it could be easier to use named colors (`white`) or hex-values (`#fff`). 😊